apps/web/vibes/soul/form/select/index.tsx

```diff
@@ -51,7 +52,7 @@ export function Select({
           onBlur={onBlur}
           onFocus={onFocus}
         >
-          <SelectPrimitive.Value placeholder={placeholder} />
+          <SelectPrimitive.Value aria-label={value} placeholder={placeholder} />
```

Comment:
Feels weird to have the `aria-label` as the value here since the value might not be a user-readable string

Reply:
I agree, I don't think we need this one here, only the one in the `Trigger` component.